src/main/java/com/gtnewhorizons/angelica/config/AngelicaConfig.java

```diff
@@ -23,6 +23,11 @@ public class AngelicaConfig {
     @Config.RequiresMcRestart
     public static boolean injectQPRendering;
 
+    @Config.Comment("Enable Angelica's test blocks")
```

Comment:
I think this would be more fitting as a JVM arg since it's intended for development.